src/operator/controllers/vizier_controller.go

```diff
@@ -126,6 +127,34 @@ func getLatestVizierVersion(ctx context.Context, client cloudpb.ArtifactTrackerC
 	return resp.Artifact[0].VersionStr, nil
 }
 
+// validCSIDriver checks if the user is running an EKS cluster, and if so, whether they have
+// the CSIDriver enabled such that persistent volumes can be deployed.
+func validCSIDriver(clientset *kubernetes.Clientset, k8sVersion string) (bool, error) {
+	// This check only needs to be done for eks clusters with K8s version > 1.22.0.
+	if !strings.Contains(k8sVersion, "-eks-") {
+		return true, nil
+	}
```

Comment:
maybe as a followup we should consider pulling this detection logic and using something that we share with https://github.com/pixie-io/pixie/blob/main/src/pixie_cli/pkg/utils/checks.go#L130-L154